src/test/java/nl/ordina/jobcrawler/service/SkillServiceTest.java

```diff
@@ -0,0 +1,298 @@
+package nl.ordina.jobcrawler.service;
+
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.repo.SkillRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+
+class SkillServiceTest {
+
+    private SkillRepository skillRepositoryMock;
+    private SkillService skillService;
+
+    private Vacancy vacancy;
+    private List<Vacancy> vacanciesDB;  // fake "database" for vacancy
+    private String skillName;
+    private Skill skill;
+    private List<Skill> skillDB;    // fake "database" for skill
+
+    @BeforeEach
+    void init() {
+        this.skillRepositoryMock = Mockito.mock(SkillRepository.class);
+        this.skillService = new SkillService(skillRepositoryMock);
```

Comment:
Could be done in @BeforeAll

Reply:
done in https://github.com/mostafa-221/jobcrawler-backend/pull/17/commits/9faea83a189e30e11f4c30d369e0f297485faeaf